tools/check_examples/README.md

```diff
@@ -20,10 +23,20 @@ To install the script:
     alias check='pipenv run python check.py'
```

Comment:
this isn't technically part of this PR but maybe it would be worth changing this to a command that will work for all users (like those using command prompt, if that is the CLI that most people use)

Reply:
or maybe add a separate option for windows users? I appreciate the convenience of the `check` alias

Reply:
I changed it to use PDM instead, so all commanes in the `README.md` starts with `pdm check` which works on all operating systems.